MantidQt/CustomInterfaces/src/Tomography/TomographyIfacePresenter.cpp

```diff
@@ -134,7 +150,10 @@ void TomographyIfacePresenter::notify(
 }
 
 void TomographyIfacePresenter::processSystemSettingsUpdated() {
-  m_model->updateSystemSettings(m_view->systemSettings());
+  m_model->setSystemSettings(m_view->systemSettings());
+
+  // update the tool information with the new system settings
+  setupConfigDialogSettingsAndUpdateModel(m_configDialog.get());
```

Comment:
Now only this method takes a pointer, the other take references, my goal with this is to centralise the `nullptr` check that is necessary to prevent a crash on initialisation, and avoid a `*` dereference on each call